### ISSUE TITLE
AttributeError: 'Time' object has no attribute 'unmasked'

### ISSUE THREAD

**Issue by ASainio** [pkgw/daschlab#17]
Ahoy, I am running RY Cnc example on Windows and local Jupyter lab. I installed everything from conda-requirements.txt and followed DASCHlab installation instructions. 

The notebook works just like it, should, including WWT, until I get attribute error below. I feel this is something super simple but I can't figure it out? Ideas?

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[23], line 2
      1 # This plate_local_id looks good ...
----> 2 sess.plates().show(13958)

File ~\AppData\Roaming\Python\Python311\site-packages\daschlab\plates.py:916, in Plates.show(self, plate_ref)
    913 if il is not None:
    914     return il  # TODO: bring it to the top, etc?
--> 916 fits_relpath = self._sess().cutout(plate)
    917 if fits_relpath is None:
    918     from . import InteractiveError

File ~\AppData\Roaming\Python\Python311\site-packages\daschlab\__init__.py:705, in Session.cutout(self, plate_ref)
    703 h["DATE-OBS"] = plate["obs_date"].fits
    704 h["MJD-OBS"] = plate["obs_date"].mjd
--> 705 h["DATE-SCN"] = plate["scan_date"].unmasked.fits
    706 h["MJD-SCN"] = plate["scan_date"].unmasked.mjd
    707 h["DATE-MOS"] = plate["mos_date"].unmasked.fits

File C:\ProgramData\anaconda3\Lib\site-packages\astropy\time\core.py:1671, in TimeBase.__getattr__(self, attr)
   1664         raise ScaleValueError(
   1665             f"Cannot convert {self.__class__.__name__} with scale "
   1666             f"'{self.scale}' to scale '{attr}'"
   1667         )
   1669 else:
   1670     # Should raise AttributeError
-> 1671     return self.__getattribute__(attr)

AttributeError: 'Time' object has no attribute 'unmasked'
```

**Comment by ASainio** [comment]
Actually I edited the __init__.py:

```
h["DATE-OBS"] = plate["obs_date"].fits
h["MJD-OBS"] = plate["obs_date"].mjd
h["DATE-SCN"] = plate["scan_date"].unmasked.fits  # These lines causes the issue
h["MJD-SCN"] = plate["scan_date"].unmasked.mjd  # These lines causes the issue
h["DATE-MOS"] = plate["mos_date"].unmasked.fits  # These lines causes the issue
```

to: 

```
> h["DATE-OBS"] = plate["obs_date"].fits
> h["MJD-OBS"] = plate["obs_date"].mjd
> h["DATE-SCN"] = plate["scan_date"].fits  
> h["MJD-SCN"] = plate["scan_date"].mjd  
> h["DATE-MOS"] = plate["mos_date"].fits  
```

And got it working. Let's hope this won't create more issues.

**Comment by pkgw** [comment]
Thanks for reporting this. I'm glad that you were able to work around the issue but it's definitely worth trying to figure out the root of the problem.

I would think that maybe you're using an unexpected version of Astropy, but if you installed things from the conda-requirements.txt file, that shouldn't be an issue.

Did you encounter this when running the RY Cnc tutorial exactly as-is, or did you make modifications / extensions to the sample notebook?

**Comment by ASainio** [comment]
Hi Peter,
Thanks for your time.

**I was following strict instructions from the tutorial without editing it using Astropy version is 5.3.4**  but other mismatches can be possible. 

It indeed looks like a version mitch-match. However, as there are not yet other issues/tickets, and I got it solved I suggest just letting it be unless others counter similar issue and they can also see my solution if it might help. 

I was able to edit my code to get cuts / filter data and get anything I need for future. 

On a side: 
This is great data and priceless to get it to use. ...The amount of work getting this data done and then almost getting it all destroyed, and then another huge work for getting them scanned and online huh.. Huge respect!.
